recipes/watcher/all/test_v1_package/conanfile.py

```diff
@@ -0,0 +1,16 @@
+from conans import ConanFile, CMake
+
+
+class TestPackageV1Conan(ConanFile):
+    settings = "os", "arch", "compiler", "build_type"
+    generators = "cmake", "cmake_find_package_multi"
+
+    def build(self):
+        cmake = CMake(self)
+        cmake.configure()
+        cmake.build()
+
+    def test(self):
+        # don't execute test_package
```

Comment:
You use subprocess.run and pass a timeout: https://docs.python.org/3/library/subprocess.html#subprocess.run

Reply:
@uilianries Thanks! I will try.

Reply:
@uilianries 
I have finished.
But watcher released newer version and updated example which uses thread.

I will rewrite test_package.cpp to follow upstream example.